test/integration_system_test_helpers.rb

```diff
@@ -0,0 +1,60 @@
+module IntegrationSystemTestHelpers
+  include Capybara::DSL
+  include Capybara::Screenshot::MiniTestPlugin
```

Comment:
We can probably ditch this, systemtests are the ones we want to use this on. We may be able to ditch the gem entirely if it's included in systemtest somewhere automagically.